modules/460-log-shipper/crds/cluster-log-destination.yaml

```diff
@@ -396,6 +404,62 @@ spec:
                           type: boolean
                           default: true
                           description: Validate the TLS certificate of the remote host.
+                splunk:
+                  type: object
+                  required:
+                  - token
+                  - endpoint
+                  properties:
+                    endpoint:
+                      type: string
+                      description: The base URL of the Splunk instance.
+                      x-examples:
+                      - "https://http-inputs-hec.splunkcloud.com"
+                    token:
+                      type: string
+                      description: Default Splunk HEC token. If an event has a token set in its metadata, it will prevail over the one set here.
```

Comment:
```suggestion
                      description: Default Splunk HEC token. If an event has a token set in its metadata, it will have priority over the one set here.
```

Reply:
Will be applied in a separate PR.